src/DynamoSandbox/Program.cs

```diff
@@ -84,21 +88,13 @@ private static void MakeStandaloneAndRun(string commandFilePath, out DynamoViewM
             var preloaderLocation = string.Empty;
             PreloadShapeManager(ref geometryFactoryPath, ref preloaderLocation);
 
-            // TODO(PATHMANAGER): Do we really libg_xxx folder on resolution path?
-            // If not, PathResolver will be completely redundant so please remove it.
-            var pathResolver = new PathResolver(preloaderLocation);
-
-            DynamoModel.RequestMigrationStatusDialog += MigrationStatusDialogRequested;
-
```

Comment:
The above `TODO` comment is outdated, the `PathResolver` is now mandatory because it needs to specify preloaded libraries.